src/components/OptionList/OptionList.tsx

```diff
@@ -212,6 +198,25 @@ function createNormalizedOptions(
   ];
 }
 
+function isSection(arr: Descriptor[]): arr is SectionDescriptor[] {
```

Comment:
This is a function for `arr is SectionDescriptor[]` which enables some type-safe features from typescript